app/src/main/java/com/peacecorps/pcsa/safety_tools/BystanderInterventionCommonFragment.java

```diff
@@ -0,0 +1,39 @@
+package com.peacecorps.pcsa.safety_tools;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.text.Html;
+import android.text.method.ScrollingMovementMethod;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.peacecorps.pcsa.R;
+
+/*
+ * Common Fragment for all 3 screens of ByStander Intervention
```

Comment:
3 subtopics under Bystander intervention. Since they followed similar UI. I chose not to write 3 fragments and inflating them. Instead the current fragment acts like a placeholder. Everytime, one of those 3 buttons is clicked, required data is sent as argument in a bundle into the fragment and populated